lib/Build.js

```diff
@@ -160,7 +160,12 @@ class Build extends Entity {
     // Returns:                 Nothing
     run(options) {
         const deviceGroup = new DeviceGroup(options);
-        this._deploy(options).
+             this._deploy(options).
```

Comment:
Just check the indentation here and elsewhere, ie. line 163 should align with 162, and the blocks at 163 and 169 should have be indented from those. Makes code more readable later.